src/python/pants/engine/native.py

```diff
@@ -452,12 +452,6 @@ def extern_call(self, context_handle, func, args_ptr, args_len):
     args = tuple(c.from_value(arg[0]) for arg in self._ffi.unpack(args_ptr, args_len))
     return self.call(c, runnable, args)
 
-  @_extern_decl('PyResult', ['ExternContext*', 'uint8_t*', 'uint64_t'])
```

Comment:
Maybe leave a `NB: We can use eval and this is how it was implemented <link to this PR>` here or somewhere else, to make it easy to re-add later? Not sure it's worth, but just a thought.

Reply:
People can look through the git history if they think we may have implemented it before and want to re-add it :)